libs/lib-a/settings.gradle.kts

```diff
@@ -1 +0,0 @@
-rootProject.name = "lib-a"
```

Comment:
I've never seen a project with multiple settings.gradle.kts files. Usually there is only one for the entire repo, AFAIK.